<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dygma Raise Assembly Tests</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="image.qrc">:/images/Dygma_logo_color.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="update_side_fw">
         <attribute name="title">
          <string>Update Side Firmware</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="QPushButton" name="verify_right">
              <property name="text">
               <string>Verify Right Side current firmware</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="flash_right">
              <property name="text">
               <string>Flash Right Side new firmware</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="verify_left">
              <property name="text">
               <string>Verify Left Side current firmware</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="flash_left">
              <property name="text">
               <string>Flash Left Side new firmware</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="log_messages">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDygma_Raise_Assembly_Tests">
    <property name="title">
     <string>log</string>
    </property>
    <addaction name="actionCopy_log"/>
    <addaction name="actionClear_log"/>
   </widget>
   <addaction name="menuDygma_Raise_Assembly_Tests"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCopy_log">
   <property name="text">
    <string>copy log</string>
   </property>
  </action>
  <action name="actionClear_log">
   <property name="text">
    <string>clear log</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
